addon/mixins/dom.js

```diff
@@ -184,7 +184,7 @@ export default Mixin.create({
       /*
        * Attach the listener and cache the listener for teardown
        */
-      element.on(eventName, coalescedCallback);
+      element.addEventListener(eventName, coalescedCallback);
```

Comment:
Unfortunately, all browsers don't support an options POJO for the last arg to `addEventListener` so we may have to feature detect (one time at the top of the module).  MDN has an example of how to detect:

```js
const PASSIVE_SUPPORTED = (() => {
  let ret = false;

  try {
    let options = Object.defineProperty({}, "passive", {
      get: function() {
        ret = true;
      }
    });

    window.addEventListener("test", null, options);
  } catch(err) {}
  return ret;
})();
```